pinc/DPage.inc

```diff
@@ -816,13 +822,14 @@ function _project_adjust_n_pages($projectid, $adjustment)
         SET n_pages = %s
```

Comment:
This is done so we can let MySQL do the arithmetic operation for us. For example, one of the caller passes "+1"  for `$adjustment`. This will translate to the following valid SQL statement:

```
UPDATE projects SET n_pages = n_pages + 1 where projectid=...
```

If we didn't do this, we would need to do a SELECT (to get the current value of `n_pages`), calculate the new value then sent the UPDATE.... but this would open us to a race condition if there are concurrent edits on the same row. The solution would be to lock the row during the read-modify-write but I don't think that's worth it as it brings its own issues.